react/react-training/src/app/screens/Game/components/GameViewer/index.js

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from 'react';
```

Comment:
The refactor is fine! but isn't necessary to extract it to another component. Usually we extract the dumb component to a file `layout.js` and the smart to an `index.js` file, in the same component folder, in this case the `Game` folder and this file will be `layout.js`. 
You can check that here https://github.com/Wolox/tech-guides/blob/master/react/style-guide.md#folder-structure